tests/src/test/java/io/streamnative/pulsar/handlers/kop/KafkaListenerNameTest.java

```diff
@@ -117,11 +117,13 @@ public void testMetadataRequestForMultiListeners() throws Exception {
                 Assert.assertEquals(partitionMetadataList.size(), numPartitions);
                 for (int i = 0; i < numPartitions; i++) {
                     final PartitionMetadata partitionMetadata = partitionMetadataList.get(i);
-                    Assert.assertEquals(partitionMetadata.error(), Errors.NONE);
-                    Assert.assertEquals(partitionMetadata.leader().host() + ":" + partitionMetadata.leader().port(),
+                    Assert.assertEquals(partitionMetadata.error, Errors.NONE);
+                    Node leader = metadataResponse.brokersById().get(partitionMetadata.leaderId.get());
+                    Assert.assertEquals(leader.host() + ":" + leader.port(),
                             expectedAddress);
                 }
             } catch (Exception e) {
+                log.error("Error", e);
```

Comment:
```suggestion
```

Don't print the whole exception stack if it's expected.

Reply:
done